### PR TITLE
Some suggestions to pow description

### PR DESCRIPTION
**Pull request by dimxy**

### CODE REVIEW

File: docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md [hunk 1]
```diff
@@ -174,23 +174,23 @@ Let us suppose that the husband is in Asia and the wife is in the Americas. Both
...
 
 ##### An Internal Conflict of Interest Arises Within the Bitcoin Network
 
 Note the paradox here. There are now two versions of the Bitcoin history that are valid, yet different.
 
-These two versions make their way across the Internet, around the world, each to the other side. When the competing messages arrive, the Bitcoin protocol sees that there is a conflict: the same money was spent twice.
+These two versions make their way across the Internet, around the world, each to the other side. When the competing messages arrive, the Bitcoin protocol sees two conflicting chain histories: the same money was spent twice in each of them.
```
Comment (smk762) [inline docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md:185]: sounds like 4 tx total.
maybe something like "...was spent twice (once on each competing chain)"

Comment (dimxy) [reply docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md]: > sounds like 4 tx total. maybe something like "...was spent twice (once on each competing chain)"

fixed, thanks

File: docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md [hunk 2]
```diff
@@ -174,23 +174,23 @@ Let us suppose that the husband is in Asia and the wife is in the Americas. Both
...
+The Longest Chain Rule simply declares that whichever of the two competing blockchains grows longer first, wins. The consensus mechanism deactivates the other version and chain users won't see it anymore.
 
-Let us suppose that the Chinese mining equipment is superior in this instance, and the Chinese miner manages to discover the next valid block hash and send out the signal before the Washington state miner can do likewise. Across the world, the moment the information arrives that the Chinese miner completed yet another valid block, the Bitcoin protocol erases the Washington state miner’s version of the Bitcoin history.
+Let us suppose that the Chinese mining equipment is superior in this instance, and the Chinese miner manages to discover the next valid block hash and send out a message with the new block before the Washington state miner can do likewise. Across the world, the moment the message with the block that Chinese miner completed, the Bitcoin protocol will select the best chain between both Chineese miner's and the Washington state miner’s versions of the Bitcoin history, based on more proof-of-work in Chineese miner's history.
```
Comment (smk762) [inline docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md:193]: `Chineese` -> `Chinese`

**Review by smk762 — APPROVED** [review]
thanks!

**Review by gcharang — APPROVED** [review]

**Pull request merged**